modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImpl.java

```diff
@@ -1711,6 +1711,20 @@ public int compare(JpaJob job1, JpaJob job2) {
 
   }
 
+  @Override
+  public List<String> getJobPayloads(String operation) throws ServiceRegistryException {
+    EntityManager em = emf.createEntityManager();
+    try {
+      Query query = em.createNativeQuery("SELECT j.payload FROM mh_job j "
```

Comment:
With Greg's DB renaming patch merged, this is now `oc_job`. I will adjust this pull request…